### DISCUSSION TITLE
New Release for resource-discovery chart

### DISCUSSION THREAD

**Discussion by idoajfrog** [tomkerkhove/promitor#2143] [General] [answered]
Hi, 
Can you please release a new chart version with this change? - 
https://github.com/promitor/charts/commit/305beb8e40b44a11735939f6138f3200fbf52e4a
The update in on master branch, but the latest release was on Jun 23, 2022 so the change is not there yet..

Thanks!!

**Comment by tomkerkhove** [comment, marked as answer]
Can you please move this to https://github.com/promitor/charts?

**Reply by idoajfrog** [reply]
Hi, 
It look like I can't move it via 
![image](https://user-images.githubusercontent.com/110334379/193544741-f39f8188-78b6-4472-8bf9-ae2ba7c34dad.png)
and also I can mark the issue as a closed :/ 
Do you want me to create a new issue there?

**Reply by tomkerkhove** [reply]
It's OK - New chart was released :)

**Reply by idoajfrog** [reply]
Thanks for the quick fix, 
You are the best!

**Reply by tomkerkhove** [reply]
Happy to help and always looking for listed end-users on https://promitor.io/ if you are interested!
